mkdocs.yml

```diff
@@ -0,0 +1,150 @@
+site_name: Piraeus Datastore
+site_url: https://piraeus.io/docs/
+site_author: The Piraeus Maintainers
+site_description: >-
+  An easy-using cloud native datastore for Kubernetes. 100% open source, run anywhere.
+
+repo_name: piraeusdatastore/piraeus-operator
+repo_url: https://github.com/piraeusdatastore/piraeus-operator
+edit_uri: edit/v2/docs
+
+copyright: Copyright &copy; 2024 The Piraeus Maintainers
+
+nav:
+  - Home: README.md
+  - Tutorials:
+    - Tutorials: tutorial/README.md
+    - Getting Started: tutorial/get-started.md
+    - Creating Replicated Volumes: tutorial/replicated-volumes.md
+    - Creating Snapshots: tutorial/snapshots.md
+  - How-To Guides:
+    - How-To Guides: how-to/README.md
+    - Advanced Deployments:
+      - Use Piraeus Datastore with an Existing LINSTOR Cluster: how-to/external-controller.md
+      - Configure the DRBD Module Loader: how-to/drbd-loader.md
+      - Install Kernel Headers to build DRBD: how-to/install-kernel-headers.md
+    - Securing Components:
+      - Configure TLS Between LINSTOR Controller and LINSTOR Satellite: how-to/internal-tls.md
+      - Configure TLS for the LINSTOR API: how-to/api-tls.md
+      - Configure TLS for DRBD Replication: how-to/drbd-tls.md
+      - Load DRBD with SecureBoot Enabled: how-to/secure-boot.md
+    - Kubernetes Distributions:
+      - Openshift: how-to/openshift.md
+      - Talos Linux: how-to/talos.md
+      - Flatcar Container Linux: how-to/flatcar.md
+      - Microk8s: how-to/microk8s.md
+      - k0s: how-to/k0s.md
+    - Networking:
+      - Deploy Piraeus Datastore behind an HTTP Proxy: how-to/http-proxy.md
+      - Deploy a NetworkPolicy for Piraeus Datastore: how-to/network-policy.md
+      - Use the Host Network for DRBD Replication: how-to/drbd-host-networking.md
+    - Maintenance Tasks:
+      - Monitor Piraeus Datastore with Prometheus Operator: how-to/monitoring.md
+      - Keep Persistent Volume Affinity Updated with LINSTOR Affinity Controller: how-to/linstor-affinity-controller.md
+      - Restore a LINSTOR Database Backup: how-to/restore-linstor-db.md
+  - Upgrades:
+    - Upgrades: how-to/upgrade/index.md
```

Comment:
Why are the upgrade sources in `how-to` even though Upgrade is it's own section?

Reply:
I guess it was originally to align it with the "4 kinds of documentation" philosophy. I'll move it to it's own section.